eng/pipelines/coreclr/templates/build-jit-job.yml

```diff
@@ -11,6 +11,7 @@ parameters:
   timeoutInMinutes: ''
   variables: {}
   dependOnEvaluatePaths: false
+  uploadAsArtifacts: false
```

Comment:
As a suggestion, instead of Boolean here - you can use `uploadAs` variable that has two values `azureBlob` and `pipelinesArtifact` - it might make easier to understand the logic below.